src/Model/AbstractArticle.php

```
@@ -285,11 +285,16 @@ public function setValidatedAt(\DateTimeInterface $validatedAt = null): void
         $this->validatedAt = $validatedAt;
     }
 
-    public function getValidatedAt(): \DateTimeInterface
+    public function getValidatedAt(): ?\DateTimeInterface
```

At least it isn't a BC break for extension: https://3v4l.org/WlWqn

---

But instead of doing this, we could simply throw an exception and keep a non-nullable type hint:
```php
if ($this->validatedAt() === null) {
    throw new \LogicException('This article is not validated yet, you should check this by calling isValidated()')
}
```

---

Might be a problem with how the whole project is working right now, for instance if entities are used directly in forms, it will be an issue I think. So I'll just approve the PR.